instrumentation/jaxrs/jaxrs-2.0/jaxrs-2.0-resteasy-3.0/jaxrs-2.0-resteasy-3.0.gradle

```diff
@@ -14,7 +15,7 @@ muzzle {
   pass {
     group = "org.jboss.resteasy"
     module = "resteasy-jaxrs"
-    versions = "[3.5.0.Final,)"
+    versions = "[3.5.0.Final,4)"
```

Comment:
Yes and no -- since 4.0 there's no `resteasy-jaxrs` module, it's now called `resteasy-core`. This means that this line does not actually change anything and the coverage is still exactly the same -- but I've added to explicitly document that 4.0 (and newer) is not supported by this module.